backend/src/api.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime
+
+from flask import Blueprint, jsonify, request
+from sqlalchemy.orm.exc import NoResultFound
+
+from .model import Transaction, db
+
+api = Blueprint("api", __name__, url_prefix="/api")
+
+
+@api.route("/transactions/<int:transaction_id>", methods=["PUT", "GET", "DELETE"])
```

Comment:
I'd consider storage all http actions in dataclasses, especially we're defining http methods in several places (especially in bigger app) and http actions type are also used inside functions - It can be helpful with bigger app - I prefer to keep all const strings somewhere